forms/app/scdsc_investigation.properties.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": [
+    {
+      "locale": "en",
+      "content": "SCDSC Investigation"
+    },
+    {
+      "locale": "fr",
+      "content": "SCDSC Investigation"
+    }
+  ],
+  "context": {
+    "place": false,
+    "person": true,
+    "expression": "!contact || (contact.type === 'person' && user.parent.type === 'health_center' && user.role === 'chw')"
```

Comment:
Should the role be "CHW" here?

Reply:
nope should be scdsc , will update. 
NB : Since it's a task it's doesn't matter that much but will still update